src/hacbs/components/Commits/tabs/CommitDetails/CommitVisualization.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {
+  action,
+  createTopologyControlButtons,
+  defaultControlButtonsOptions,
+  ModelKind,
+  TopologyControlBar,
+} from '@patternfly/react-topology';
+import { useCommitWorkflowData } from '../../../ApplicationDetails/tabs/overview/visualization/hooks/useCommitWorkflowData';
+import { getTopologyNodesEdges } from '../../../ApplicationDetails/tabs/overview/visualization/utils/visualization-utils';
+import { layoutFactory, PipelineLayout, VisualizationFactory } from '../../../topology/factories';
+//
+import './CommitVisualization.scss';
+import { commitComponentFactory } from './CommitComponentFactory';
```

Comment:
Wonder why the linter didn't pickup the import order here

Reply:
Fixed it.